google/cloud/aiplatform/prediction/predictor.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Any
+
+
+class Predictor:
+    """Interface for Predictor class that users would be implementing."""
```

Comment:
Consider overhauling the docstrings in this class to add more detail. If the user is implementing this class, they likely need more detail than what is provided to create a proper implementation.